roles/openshift_health_checker/openshift_checks/efk_check.py

```diff
@@ -0,0 +1,88 @@
+# pylint: disable=missing-docstring
+"""
+Util functions for performiong checks on an Elasticsearch, Fluentd, and Kibana stack
+"""
+
+import json
+import os
+
+from openshift_checks import OpenShiftCheckException, get_var
+
+
+class EFKCheck(object):
+
+    @staticmethod
+    def is_first_master(task_vars):
```

Comment:
Since subclasses also inherit from OpenShiftCheck, naming this method `is_active` would cause it to be shadowed by the one defined in `OpenShiftCheck`. A solution might be to have `EFKCheck` inherit from `OpenShiftCheck`, and avoid multiple inheritance in each of the logging component classes (i.e. `logging/kibana.py` would have `EFKCheck` as its only parent, rather than both `EFKCheck` and `OpenShiftCheck`). This would also mean [altering this method to be recursive so that each check that only inherits from `EFKCheck` actually runs](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_health_checker/action_plugins/openshift_health_check.py#L72)